ITT mentoring and first post (#675)
DFE-Digital/bat-design-history: app/posts/claim-general-mentor-training-funding/2022-06-26-general-mentor-training-grant-guidance-and-claim-form.md

---
title: General mentor training grant guidance and claim form
description: Guidance page summarising ITT gerneral mentor funding and claim form.
date: 2022-06-26
---

> ## ITT market reform

Reviewer: It's a bit unclear going straight from the page title h1 to this h2 heading. Drop the heading, or add an intro para?

Author: i think its fine to have heading followed by a head, but I've removed it

> ## ITT market reform

Author: ```suggestion
```

> [The initial teacher training (ITT) market review report](https://www.gov.uk/government/publications/initial-teacher-training-itt-market-review-report) was published in July&nbsp;2021. After public consultation, the DfE published their [response to the ITT market review](https://www.gov.uk/government/consultations/initial-teacher-training-itt-market-review) was published in December&nbsp;2021.

Reviewer: ```suggestion
[The initial teacher training (ITT) market review report](https://www.gov.uk/government/publications/initial-teacher-training-itt-market-review-report) was published in July&nbsp;2021. After public consultation, the DfE published their [response to the ITT market review](https://www.gov.uk/government/consultations/initial-teacher-training-itt-market-review) in December&nbsp;2021.
```

> The recommendations will be in place for the 2024 to 2025 academic year.

Author: ```suggestion
The recommendations include new requirements for:

- mentors to train take part in 20 hours of initial mentor training and 6 hours of top-up training in subsequent years
- ITT providers to have lead mentors or a lead mentorship team
- trainees to take part in ’Intensive training and practice’ sessions (4 weeks for postgraduate courses and 6 weeks for undergraduate courses)

The recommendations will be in place for the 2024 to 2025 academic year.
```

> ### ITT reform funding

Author: ```suggestion
## ITT reform funding
```

To support providers and schools in meeting the new requirements the DfE is making grant funding available.

> For schools the funding is to help cover the cost of classroom cover while mentors train. In the 2024 to 2025 school year general mentors will need to complete 20&nbsp;hours of initial mentor training. In subsequent years general mentors who have completed the 20&nbsp;hours will need 6&nbsp;hours of training.

Author: Added

> [See full details of the ITT reform funding](https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/1077177/Initial_Teacher_Training__ITT__Reform_Funding_Guidance_160522.pdf).

Author: Done

### Claim form

![Google form asking user for their email address](google-form/01-email-address.png "Claim form — email address")

![Google form asking user for the URN of their school](google-form/02-school.png "Claim form — school unique reference number (URN)")

![Google form asking user for number of mentors, hours of classroom cover needed, and hours of classroom cover paid for by the provider](google-form/03-mentors.png "Claim form — number of mentors and training time")

![Google form asking user if each mentor trained at least one trainee](google-form/04-declaration.png "Claim form — each mentor trained at least one trainee")

![Google form confirmation the form has been completed](google-form/05-confirmation.png "Claim form — confirmation")

### Information required for a claim

#### School details

So we know who is making the claim and who to pay.

The amount of funding available is adjusted for location. To pay the correct amount DfE will need to know where the school is.

| **Location** | **Amount** |
|:-----:|:-----:|
| Inner London | £1072 |
| Outer London | £965 |
| Fringe | £902 |

> | Rest of England | £876 |

Author: Done

This also allows us to check compliance using data from [Register trainee teachers](https://www.register-trainee-teachers.service.gov.uk/) (Register). Schools can only claim for mentors who mentor at least one trainee each. Assuming Register collects placement data by the end of the 2024 to 2025 academic year then we can verify that the school has not claimed for more teachers than it is eligible to.

#### Number of general mentors

This is a basic check on the claim. Schools can claim a maximum of 20&nbsp;hours per trainee. It would also help the DfE get an understanding of how many general mentors there are.

#### Hours of classroom cover so the mentor could train

The aim of the grant is to help schools pay for cover so the mentor isn’t compelled to train out of school hours.

> To simplify the grant, schools are claiming for hours of backfill cover. Rather than specifying the costs that may have been incurred.

Reviewer: Keep this consistent with the heading?

```suggestion
To simplify the grant, schools are claiming for hours of classroom cover. Rather than specifying the costs that may have been incurred.
```

#### Hours of classroom cover paid for by 

> To prevent paying schools for something already paid for, the DfE wants to deduct any hours of backfill cover paid for by the ITT provider.

Reviewer: ```suggestion
To prevent paying schools for something already paid for, the DfE wants to deduct any hours of classroom cover paid for by the ITT provider.
```

> The form could ask for ‘hours of backfill not paid for by the provider’. My assumption is that it's easier to ask for the cover and the cover paid for by the provider separately.

Reviewer: Bit weird to introduce a "my" halfway through the article - it's not clear who this refers to.

Author: changes it to "our" for consistency

> The form could ask for ‘hours of backfill not paid for by the provider’. My assumption is that it's easier to ask for the cover and the cover paid for by the provider separately.

Author: ```suggestion
The form could ask for ‘hours of backfill not paid for by the provider’. Our assumption is that it's easier to ask for the cover and the cover paid for by the provider separately instead of asking the user to make the calculation.
```

---
title: General mentor training grant guidance and claim form

> description: Guidance page summarising ITT gerneral mentor funding and claim form.

Reviewer 2: ```suggestion
description: Guidance page summarising ITT general mentor funding and claim form.
```

> date: 2022-06-26

Reviewer 2: Why is this date so far in the past?

Author: Because I wrote this ages ago.

---

[The initial teacher training (ITT) market review report](https://www.gov.uk/government/publications/initial-teacher-training-itt-market-review-report) was published in July&nbsp;2021. After public consultation, the DfE published their [response to the ITT market review](https://www.gov.uk/government/consultations/initial-teacher-training-itt-market-review) in December&nbsp;2021.

> The recommendations which need to be followed from the 2024/25 academic year onwards, include requirements for:

Reviewer 2: ```suggestion
The recommendations which need to be followed from the 2024 to 2025 academic year onwards, include requirements for:
```

Reviewer 2: In Register we always use `YYYY to YYYY`.

- mentors to train take part in 20 hours of initial training and 6 hours of top-up training in subsequent years
- ITT providers to have lead mentors or a lead mentorship team
- trainees to take part in ’Intensive training and practice’ sessions (4 weeks for postgraduate courses and 6 weeks for undergraduate courses)

## ITT reform funding

To support providers and schools in meeting the new requirements the DfE is making grant funding available.

> For schools the funding is to help cover the cost of classroom cover while mentors train. In the 2024/25 academic year, general mentors will need to complete 20&nbsp;hours of initial mentor training. In subsequent years general mentors who have completed the 20&nbsp;hours will need 6&nbsp;hours of training.

Author: ```suggestion
For schools the funding is to help cover the cost of classroom cover while mentors train. In the 2024 to 2025 academic year, general mentors will need to complete 20&nbsp;hours of initial mentor training. In subsequent years general mentors who have completed the 20&nbsp;hours will need 6&nbsp;hours of training.
```

> [See full details of the ITT reform funding.](https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/1077177/Initial_Teacher_Training__ITT__Reform_Funding_Guidance_160522.pdf)

Reviewer 2: ```suggestion
[See full details of the ITT reform funding (PDF).](https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/1077177/Initial_Teacher_Training__ITT__Reform_Funding_Guidance_160522.pdf)
```

Do you think this should say it's going to an external site or the file size? should it force a new window?

Author: Yeah, forced new tab.

## Prototype

One of the streams of work for the mentoring team is looking into how the grants will be distributed.

We plan to distribute the funding for ITT providers through DfE’s normal business processes.

Because the general mentor training grant is available to more than 20,000&nbsp;schools and we need specific information for each claim, a digital form is necessary to collect the data.

To test our questions and assumptions about how the service will work, we built a guidance page and a form (in Google forms).

### Guidance

> ![GOV.UK page with summary of ITT general mentor training grant](general-mentor-training-grant-guidance.png "Apply for ITT general mentor training funding")

Author: ```suggestion
![GOV.UK page with summary of ITT general mentor training grant](general-mentor-training-grant-guidance.png "Apply for ITT general mentor training funding guidance page")
```

### Claim form

> ![Google form asking user for their email address](google-form/01-email-address.png "Claim form — email address")

Reviewer 2: Prefacing these captions with `Claim form –` feels redundant. Perhaps make the captions in to sentences?

> ![Google form asking user for their email address](google-form/01-email-address.png "Claim form — email address")

Author: ```suggestion
![Google form asking user for their email address](google-form/01-email-address.png "Google form asking user for their email address")
```

> ![Google form asking user for the URN of their school](google-form/02-school.png "Claim form — school unique reference number (URN)")

Author: ```suggestion
![Google form asking user for the URN of their school](google-form/02-school.png "Google form asking user for the unique reference number (URN) of their school")
```

> ![Google form asking user for number of mentors, hours of classroom cover needed, and hours of classroom cover paid for by the provider](google-form/03-mentors.png "Claim form — number of mentors and training time")

Author: ```suggestion
![Google form asking user for number of mentors, hours of classroom cover needed, and hours of classroom cover paid for by the provider](google-form/03-mentors.png "Google form asking user for number of mentors, hours of classroom cover needed, and hours of classroom cover paid for by the provider")
```

> ![Google form asking user if each mentor trained at least one trainee](google-form/04-declaration.png "Claim form — each mentor trained at least one trainee")

Author: ```suggestion
![Google form asking user if each mentor trained at least one trainee](google-form/04-declaration.png "[Google form asking user if each mentor trained at least one trainee")
```

> ![Google form confirmation the form has been completed](google-form/05-confirmation.png "Claim form — confirmation")

Author: ```suggestion
![Google form 
page showing the user confirmation the form has been completed](google-form/05-confirmation.png "Google form 
page showing the user confirmation the form has been completed")
```

### Information required for a claim

#### School details

So we know who is making the claim and who to pay.

The amount of funding available is adjusted for location. To pay the correct amount DfE will need to know where the school is.

| **Location**    | **Amount** |

> |:---------------:|:----------:|

Author: ```suggestion
|:---            :|:       ---:|
```

| Inner London    |      £1072 |
| Outer London    |       £965 |
| Fringe          |       £902 |

> | Rest of England |       £876 |

Reviewer 2: Both these columns are centred. Ideally they would be left and right aligned respectively.

Author: Yep — just learnt how to do that in markdown.

> This also allows us to check compliance using data from [Register trainee teachers](https://www.register-trainee-teachers.service.gov.uk/) (Register). Schools can only claim for mentors who mentor at least one trainee each. Assuming Register collects placement data by the end of the 2024/25 academic year then we can verify that the school has not claimed for more teachers than it is eligible to.

Author: ```suggestion
This also allows us to check compliance using data from [Register trainee teachers](https://www.register-trainee-teachers.service.gov.uk/) (Register). Schools can only claim for mentors who mentor at least one trainee each. Assuming Register collects placement data by the end of the 2024 to 2025 academic year then we can verify that the school has not claimed for more teachers than it is eligible to.
```

#### Number of general mentors

This is a basic check on the claim. Schools can claim a maximum of 20&nbsp;hours per trainee. It would also help the DfE get an understanding of how many general mentors there are.

#### Hours of classroom cover so the mentor could train

The aim of the grant is to help schools pay for cover so the mentor isn’t compelled to train out of school hours.

To simplify the grant, schools are claiming for hours of classroom cover. Rather than specifying the costs that may have been incurred.

#### Hours of classroom cover paid for by 

To prevent paying schools for something already paid for, the DfE wants to deduct any hours of classroom cover paid for by the ITT provider.

The form could ask for ‘hours of backfill not paid for by the provider’. Our assumption is that it's easier to ask for the cover and the cover paid for by the provider separately instead of asking the user to make the calculation.

#### Declaration that each mentor mentored at least one trainee each

> Funding is only available to mentors who mentored a trainee in the 2024/25 academic year. The minimum ratio of trainees to mentors is 1:1. For example, if a trainee had 2 mentors and neither of those mentors mentored another trainee, then the school could only claim for one of those mentors.

Author: ```suggestion
Funding is only available to mentors who mentored a trainee in the 2024 to 2025 academic year. The minimum ratio of trainees to mentors is 1:1. For example, if a trainee had 2 mentors and neither of those mentors mentored another trainee, then the school could only claim for one of those mentors.
```

#### Email address of user

So the DfE can contact the user about their claim.

### Assumptions and questions to test

The main things we wanted to understand were:

- participants’ reaction to new training requirement, the impact that would have and how the funding was perceived

- school’s use of cover and anticipated costs

- participants’ understanding and reaction to the rules of the funding

- what funding placement schools get from providers and if this is specifically for time when the trainee

- how participants would expect to be made aware of their entitlement to claim this funding

> - their ability to get hold of the data needed to make a claim

Reviewer 2: Is it intentional that these bullets are more spaced than the bullets further up the page?

> - participants’ reaction to new training requirement, the impact that would have and how the funding was perceived
> 
> - school’s use of cover and anticipated costs
> 
> - participants’ understanding and reaction to the rules of the funding
> 
> - what funding placement schools get from providers and if this is specifically for time when the trainee
> 
> - how participants would expect to be made aware of their entitlement to claim this funding
> 
> - their ability to get hold of the data needed to make a claim

Author: ```suggestion
- participants’ reaction to new training requirement, the impact that would have and how the funding was perceived
- school’s use of cover and anticipated costs
- participants’ understanding and reaction to the rules of the funding
- what funding placement schools get from providers and if this is specifically for time when the trainee
- how participants would expect to be made aware of their entitlement to claim this funding
- their ability to get hold of the data needed to make a claim
```

## Summary of findings

> If your email address ends ‘digital.education.gov.uk’, you can see the [analysis of the research](https://docs.google.com/presentation/d/1wcgMn0JjYUmSj801qqXHzm4kQo11QLHE5LSQjD691po/edit#slide=id.g1221b5a2b76_1_1036).

Reviewer 2: I find this sentence really confusing. It took me several reads to understand what you meant. I thought the research was on the topic of having email addresses ending in x! 

Perhaps front load with the research and add `(only viewable by those with emails ending in x)`

> If your email address ends ‘digital.education.gov.uk’, you can see the [analysis of the research](https://docs.google.com/presentation/d/1wcgMn0JjYUmSj801qqXHzm4kQo11QLHE5LSQjD691po/edit#slide=id.g1221b5a2b76_1_1036).

Author: ```suggestion
You can see the [analysis of the research] (https://docs.google.com/presentation/d/1wcgMn0JjYUmSj801qqXHzm4kQo11QLHE5LSQjD691po/edit#slide=id.g1221b5a2b76_1_1036) (only viewable by those with emails ending in ‘digital.education.gov.uk’).
```
